tsdb/index/postings.go

```diff
@@ -428,6 +428,13 @@ func EmptyPostings() Postings {
 	return emptyPostings
 }
 
+// IsEmptyPostings returns true if the postings are an empty postings list.
+// When this function returns false, it doesn't mean that the postings isn't empty
+// (it could be an empty intersection of two non-empty postings, for example).
+func IsEmptyPostings(p Postings) bool {
```

Comment:
The function name can be confusing. How about this? Or something that indicates that it can be not empty
```suggestion
func IsEmptyPostingsType(p Postings) bool {x
```

Reply:
Sure, thank you for the suggestion, applied here:
https://github.com/grafana/mimir-prometheus/pull/377/commits/e246ce436d69cd48334b0e85d25a8225e8110e45